packages/connect/src/api/showDeviceTutorial.ts

```diff
@@ -0,0 +1,24 @@
+import { AbstractMethod } from '../core/AbstractMethod';
+import { getFirmwareRange } from './common/paramsValidator';
+import { PROTO } from '../constants';
+import { UI } from '../events';
+
+export default class ShowDeviceTutorial extends AbstractMethod<
+    'showDeviceTutorial',
+    PROTO.CancelAuthorization
```

Comment:
this is some leftover ins't it?

Reply:
https://github.com/trezor/trezor-suite/pull/8964/commits/388ff76d53da7bfe8baccfa55e3f5b294fd0bdbb